src/pkg/services/m365/api/users_test.go

```diff
@@ -116,164 +113,3 @@ func (suite *UsersUnitSuite) TestEvaluateMailboxError() {
-					Reply(400).
-					JSON(parseableToMap(t, odErrMsg("anycode", string(graph.NoSPLicense))))
-				interceptV1Path("users", "user", "mailFolders", api.MailInbox).
-					Reply(400).
-					JSON(parseableToMap(t, odErr(string(graph.ResourceNotFound))))
-			},
-			expectErr: func(t *testing.T, err error) {
-				assert.NoError(t, err, clues.ToCore(err))
-			},
-		},
-		{
-			name: "default drive err - other error",
-			setGocks: func(t *testing.T) {
-				interceptV1Path("users", "user", "drive").
-					Reply(400).
-					JSON(parseableToMap(t, odErrMsg("somecode", "somemessage")))
-			},
-			expectErr: func(t *testing.T, err error) {
-				assert.Error(t, err, clues.ToCore(err))
-			},
-		},
-		{
-			name: "mail inbox err - user not found",
-			setGocks: func(t *testing.T) {
-				interceptV1Path("users", "user", "drive").
-					Reply(200).
-					JSON(parseableToMap(t, models.NewDrive()))
-				interceptV1Path("users", "user", "mailFolders", api.MailInbox).
-					Reply(400).
-					JSON(parseableToMap(t, odErr(string(graph.RequestResourceNotFound))))
-			},
-			expectErr: func(t *testing.T, err error) {
-				assert.ErrorIs(t, err, graph.ErrResourceOwnerNotFound, clues.ToCore(err))
-			},
-		},
-		{
-			name: "mail inbox err - user not found",
-			setGocks: func(t *testing.T) {
-				interceptV1Path("users", "user", "drive").
-					Reply(200).
-					JSON(parseableToMap(t, models.NewDrive()))
-				interceptV1Path("users", "user", "mailFolders", api.MailInbox).
-					Reply(400).
-					JSON(parseableToMap(t, odErr(string(graph.MailboxNotEnabledForRESTAPI))))
-			},
-			expectErr: func(t *testing.T, err error) {
-				assert.NoError(t, err, clues.ToCore(err))
-			},
-		},
-		{
-			name: "mail inbox err - authenticationError",
-			setGocks: func(t *testing.T) {
-				interceptV1Path("users", "user", "drive").
-					Reply(200).
-					JSON(parseableToMap(t, models.NewDrive()))
-				interceptV1Path("users", "user", "mailFolders", api.MailInbox).
-					Reply(400).
-					JSON(parseableToMap(t, odErr(string(graph.AuthenticationError))))
-			},
-			expectErr: func(t *testing.T, err error) {
-				assert.NoError(t, err, clues.ToCore(err))
-			},
-		},
-		{
-			name: "mail inbox err - other error",
-			setGocks: func(t *testing.T) {
-				interceptV1Path("users", "user", "drive").
-					Reply(200).
-					JSON(parseableToMap(t, models.NewDrive()))
-				interceptV1Path("users", "user", "mailFolders", api.MailInbox).
-					Reply(400).
-					JSON(parseableToMap(t, odErrMsg("somecode", "somemessage")))
-			},
-			expectErr: func(t *testing.T, err error) {
-				assert.Error(t, err, clues.ToCore(err))
-			},
-		},
-	}
-	for _, test := range table {
-		suite.Run(test.name, func() {
-			t := suite.T()
-			ctx, flush := tester.NewContext(t)
-
-			defer flush()
-			defer gock.Off()
-
-			test.setGocks(t)
-
-			_, err := suite.its.gockAC.Users().GetInfo(ctx, "user")
-			test.expectErr(t, err)
-		})
-	}
-}
-
-func (suite *UsersIntgSuite) TestUsers_GetInfo_quotaExceeded() {
```

Comment:
Same here. Added a unit test in an older PR at ^TestEnabledUnitSuite/TestGetMailboxInfo